<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCustomPlot" name="showPic" native="true"/>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="Text_Show" native="true">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="Port_set"/>
    <addaction name="actionduodaoshezhi"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>测量</string>
    </property>
    <addaction name="actiontanceqi"/>
    <addaction name="actionyuanchengshezhi"/>
    <addaction name="actionkaishiceliang"/>
    <addaction name="actiontingzhiceliang"/>
    <addaction name="actionqingpu"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="actioncengdiepailie"/>
    <addaction name="actionshuipingpingpu"/>
    <addaction name="actionchuizhipingpu"/>
    <addaction name="separator"/>
    <addaction name="actionpubutu"/>
    <addaction name="actionjishulvquxian"/>
    <addaction name="actionjilianglvquxian"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>0</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="B_LoopStart"/>
   <addaction name="B_LoopEnd"/>
  </widget>
  <action name="Port_set">
   <property name="text">
    <string>串口设置</string>
   </property>
  </action>
  <action name="B_LoopStart">
   <property name="text">
    <string>开始采集</string>
   </property>
  </action>
  <action name="B_LoopEnd">
   <property name="text">
    <string>停止采集</string>
   </property>
  </action>
  <action name="actiontanceqi">
   <property name="text">
    <string>探测器</string>
   </property>
  </action>
  <action name="actionyuanchengshezhi">
   <property name="text">
    <string>远程设置</string>
   </property>
  </action>
  <action name="actionkaishiceliang">
   <property name="text">
    <string>开始测量</string>
   </property>
  </action>
  <action name="actiontingzhiceliang">
   <property name="text">
    <string>停止测量</string>
   </property>
  </action>
  <action name="actionqingpu">
   <property name="text">
    <string>清谱</string>
   </property>
  </action>
  <action name="actioncengdiepailie">
   <property name="text">
    <string>层叠排列</string>
   </property>
  </action>
  <action name="actionshuipingpingpu">
   <property name="text">
    <string>水平排列</string>
   </property>
  </action>
  <action name="actionchuizhipingpu">
   <property name="text">
    <string>垂直平铺</string>
   </property>
  </action>
  <action name="actionpubutu">
   <property name="text">
    <string>瀑布图</string>
   </property>
  </action>
  <action name="actionjishulvquxian">
   <property name="text">
    <string>计数率曲线</string>
   </property>
  </action>
  <action name="actionjilianglvquxian">
   <property name="text">
    <string>剂量率曲线</string>
   </property>
  </action>
  <action name="actionduodaoshezhi">
   <property name="text">
    <string>多道设置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
